packages/app/source/shared/collections/open-appeals.js

```diff
@@ -0,0 +1 @@
+Donations.OpenAppeals = new Mongo.Collection('donations_open_appeals');
```

Comment:
The separate collection is still great for exposing different properties for the public facing appeals etc.